test/hotspot/jtreg/vmTestbase/nsk/jdi/ExceptionEvent/_itself_/exevent008.java

```diff
@@ -80,7 +80,7 @@ public class exevent008 {
     private ExceptionRequest eRequest;
     private ReferenceType rType, rTypeEx;
     private String cmd;
-    private int counter1 = 0, counter2 = 0;
+    private volatile int counter1 = 0, counter2 = 0;
```

Comment:
The volatile is not enough for increment. It might be not affect test if you check 0/non-0 only but still not a good way.
Better to use AtomicInteger or set/read variable only.

Reply:
Thanks for explanation. Can you please add corresponding comment. The volatile counters are rare thing...